src/index.ts

```diff
@@ -57,15 +57,20 @@ program
   // Stops transpilation after the specified pass
   .option('--until <pass>')
   .option('--no-warnings')
-  .action((file: string, options: CliOptions) => {
-    if (!isValidSolFile(file)) return;
-    try {
-      transpile(compileSolFile(file, options.warnings), options).map(([name, cairo, abi]) => {
-        outputResult(name, cairo, options, '.cairo', abi);
-      });
-    } catch (e) {
-      handleTranspilationError(e);
-    }
+  .action((files: string[], options: CliOptions) => {
+    if (files.some((file) => !isValidSolFile(file))) return;
```

Comment:
Shouldn't we check if we every sol file is valid, if not we should print an error about the invalid ones

Reply:
As in give errors for all failing ones instead of fast failing as we do here?